install/0000_30_machine-api-operator_11_deployment.yaml

```diff
@@ -64,8 +64,8 @@ spec:
           value: "8080"
         resources:
           requests:
-            cpu: 10m
-            memory: 50Mi
+            cpu: 50m
```

Comment:
why did we need to bump this, did you see anything particular concerning?

Reply:
There are random flakes in CI, which happen only for this PR. This is the only place where the difference could be made. I'll let the CI pass, before removing the commit or adjusting it to a lower value.

Reply:
>There are random flakes in CI

Have you seen logs in CI that shows this deployment is needing more resources? are there logs on CI for this pod getting throttled because of lack of resources?

Reply:
I'll have a look again, but no I found none such yet.